### ISSUE TITLE
.

### ISSUE THREAD

**Issue by hosseinaskari-cs** [yang-song/score_sde_pytorch#34]

**Comment by hosseinaskari-cs** [comment]
I tried thousands of times with different strategies on my machine. The code does not work on my machine. It does not show me any error. But it gets stock at the beginning and shows no output. I ran the code on Google Colab, it is working. So the problem might be with my machine. I am using an RTX 3090 with 64 Gibabayte Ram. Please let me know what the problem can be. I installed different versions of PyTorch, but none of them is going to work. Please give me a suggestion. Note that other codes I already tried on my machine and the same environment, and they are working as well. 

python main.py --config configs/ve/cifar10_ddpm.py --eval_folder eval --mode train --workdir work_dir

**Comment by 42Shawn** [comment]
I have gone through the same issue. Have you solved it? I debugged the code, and found the whole program is stuck at loss.backward().

**Comment by hosseinaskari-cs** [comment]
> I have gone through the same issue. Have you solved it? I debugged the code and found the whole program is stuck at loss. backward().

Not yet actually. Unforatently, I could not find any PyTorch code of the score-based models except on this page.
